ui/src/pages/help-center/HelpCenter.constants.ts

```diff
@@ -0,0 +1,110 @@
+export const cardIds = {
+  pmmDocs: 'pmm-docs',
+  support: 'support',
+  forum: 'forum',
+  pmmDump: 'pmm-dump',
+  pmmLogs: 'pmm-logs',
+  tips: 'tips',
+};
+
+export const startIcon = {
+  download: 'download',
+  map: 'map',
+};
+
+export const CardsData = [
+  {
+    id: cardIds.pmmDocs,
+    title: 'PMM Documentation',
+    description:
+      'From setup to troubleshooting, you’ll find step-by-step instructions, tips, and best practices to get the most out of PMM.',
+    buttons: [
+      {
+        buttonText: 'View docs',
```

Comment:
Since this prop is is specific to buttons, I don't it needs the `button` prefix (can be either text/content)